browser/playlist/playlist_page_handler.cc

```diff
@@ -62,12 +85,12 @@ void PlaylistPageHandler::GetPlaylist(
     std::move(callback).Run(nullptr);
     return;
   }
-
   std::vector<mojo::StructPtr<playlist::mojom::PlaylistItem>> items;
   for (const auto& item : playlist->items) {
     items.push_back(playlist::mojom::PlaylistItem::New(
-        item.id, item.title, GURL(item.page_src), GURL(item.media_file_path),
-        GURL(item.thumbnail_path), item.media_file_cached));
+        item.id, item.title, GURL(item.page_src),
+        GURL("file://" + item.media_file_path),
+        GURL("file://" + item.thumbnail_path), item.media_file_cached));
```

Comment:
do we need that change for Android only or desktop uses `file://` as well?

Reply:
I think this change doesn't affect for desktop because desktop playlist page UI doesn't use this path.
Instead, it fetches media file or thumbnail file by using `item.id` via `PlaylistDataSource` with below urls.
```
 chrome-untrusted://playlist-data/' + itemId + '/thumbnail/
 chrome-untrusted://playlist-data/' + itemId + '/media'
```
So, I think this change will be fine. WDYT @sangwoo108 ?

Reply:
Yeah, this changes are also what I made through with @deeppandya . Technically, file:// scheme should be added regardless of platform, I think. And that shouldn't affect Desktop's behavior like @simonhong said.

Reply:
Today I realized that this isn't good place to add "file://" as item.foo_path can be http:// . Maybe we should do this when caching is done.

Reply:
Not sure but maybe @deeppandya added `file://` to get valid `GURL`.
When I passed local file path w/o file scheme, GURL was invalid one.

Reply:
> Not sure but maybe @deeppandya added file:// to get valid GURL.

Yep, exactly. This was added while @deeppandya and I were pairprogramming.

Reply:
btw, I think android also could use `chrome-untrusted://playlist-data/' + itemId + '/media'` as `PlaylistDataSource` is not desktop specific instead of local file path?

Reply:
I'm not sure cuz I think I heard that Android uses external media player library.